### PR TITLE
DMVP-1262: Add ability to push files from local to remote with initial commit, and PR tit…

### PR DESCRIPTION
**Pull request by VahagnMian**

### CODE REVIEW

**Review by 0katrinpetrosyan0 — COMMENTED** [review]
nenc ara es nerqevi karmir gcery chlini, 
repository.tf-i mej prevent_destroy-y true petq a lini
mekel aramy asec vor pre-commit yamly kara chlini  vor PR-i tak comment a grum

**Review by 0katrinpetrosyan0 — COMMENTED** [review]
branchi vra el conflict ka 
mekel variable-nery vor grum es ete ham description ham type grum es bolori dzevy nuyn dzev pahi orinaki pes
variable "add_precommit" {
  description  = "Add precommit configuration to repo's files"
  type            = bool
  default        = false
}

File: modules/github-repository/resources/deleteme [hunk 1]
```diff
@@ -0,0 +1 @@
+DMVP-1097: Add sso-rbac module
```
Comment (mrdntgrn) [inline modules/github-repository/resources/deleteme:1]: seems no need for this here

File: modules/github-repository/README.md [hunk 2]
```diff
@@ -189,20 +191,23 @@ No modules.
...
-| [github_branch_protection_v3.branch_protection](https://registry.terraform.io/providers/hashicorp/github/latest/docs/resources/branch_protection_v3) | resource |
-| [github_repository.repository](https://registry.terraform.io/providers/hashicorp/github/latest/docs/resources/repository) | resource |
-| [github_repository_collaborator.collaborator](https://registry.terraform.io/providers/hashicorp/github/latest/docs/resources/repository_collaborator) | resource |
-| [github_repository_webhook.repository_webhook](https://registry.terraform.io/providers/hashicorp/github/latest/docs/resources/repository_webhook) | resource |
-| [github_team_repository.team_repository](https://registry.terraform.io/providers/hashicorp/github/latest/docs/resources/team_repository) | resource |
-| [github_team_repository.team_repository_by_slug](https://registry.terraform.io/providers/hashicorp/github/latest/docs/resources/team_repository) | resource |
+| [github_actions_secret.repository_secret](https://registry.terraform.io/providers/integrations/github/latest/docs/resources/actions_secret) | resource |
```
Comment (mrdntgrn) [inline modules/github-repository/README.md:194]: duplicate terraform docs in this readme file

**Review by 0katrinpetrosyan0 — APPROVED** [review]

**Pull request merged**